notebooks/example.ipynb

```diff
@@ -182,24 +139,163 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
    "id": "9fe7687f-5c6c-44fc-aaf7-92711ce97457",
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "user:\n",
-      "  name: John Doe\n",
-      "  favorite_food: burritos\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "%%bash\n",
     "cat ./fixtures/rendered_config.yaml"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea5eff92",
+   "metadata": {},
+   "source": [
+    "## Example 3: Comparing two config files\n",
+    "Let's explore using the `config.compare()` method to compare two two config files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71d27692",
+   "metadata": {},
+   "source": [
+    "We again need to start by importing the `uwtools.api.config` from the uwtools python package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7fb40514",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uwtools.api import config\n",
+    "from uwtools.api.logging import use_uwtools_logger\n",
+    "use_uwtools_logger()"
```

Comment:
Thanks for adding this! I'm executing the notebook locally, and I really like the way the output looks with the logging enabled.